### ISSUE TITLE
Improve Apache instruvtion to redirect to public

### ISSUE THREAD

**Issue by PeopleInside** [flarum/docs#46]
Hi, as for the issue https://github.com/flarum/core/issues/1689 I suggest to integrate informations on this post: https://discuss.flarum.org/d/17977-forum-public-why-not-just-forum/7

This works also on localhost, tested. The documentation given by Flarum Doc did not work or is too complicated.

SHC found the solution.

**Comment by CDK2020** [comment]
As of checking right now, the docs already specify to use the public directory. Your issue (I've been following the thread) seems like a very edge case. The docs serve to provide general help for the majority of users.

**Comment by PeopleInside** [comment]
The solution of add the code on htaccess solve the redirect issue.
If i never read the SHC comment today was not able to point the domain to public on local host staying on the doc. Doc should help Apache users to redirect from forum folder to public. Will be useful include suggestion of SHC.

In the localhost now i am using also php flarum install as the install on the browser fail and not only for me but this is another issue.

**Comment by clarkwinkelmann** [comment]
Doesn't https://flarum.org/docs/install.html#customizing-paths, as shared by Jordan on Discuss, already explain the exact recommended way to achieve that ?

**Comment by PeopleInside** [comment]
Umh for me no. I discussed a lot on the forum and was unable to point, on localhost to public.
After some week i found a user that share a simple solution, more simple that edit Apache configuration.

Just create an htaccess file that is really doing easy his job.

**Comment by franzliedke** [comment]
I agree with most replies here, and also with what @jordanjay29 mentioned in the forum topic.

That said, there is definitely room for improvement in our installation docs. We will keep this issue (and also https://github.com/flarum/core/issues/1669) open as a reminder, and will spend more time to research the best solution, also considering the one from the forum. Thanks for bringing it up in any case!

Our goals:
1. Our default will be the most secure setup, aimed at virtual or dedicated servers.
2. For people who don't have the possibility to run Flarum in such a setup, we **make them aware of the consequences** and give them explicit instructions that are easy to follow.
3. Editing the skeleton (flarum/flarum) is okay, contrary to what was posted in the linked forum topic. We plan to bring this to a point (upon stable release), where no more changes to the skeleton occurs during version updates, which means this is *your turf* and safe to change.

**Comment by PeopleInside** [comment]
Ok thanks @franzliedke . Consider i look into the forum, user post also and i test... i write here just if i found issues and think to can give to you a small but precious contribute.

No everyone is good with servers. On my VPS no issue because i have a control panel so just need to go on domain and insert the domain point to the public, all is done from my control panel.

Without this with just only the docs i will be for sure unable to point to the public folder as this happen for me on localhost install.

I believe the .htaccess solution is safe and works on localhost test so i think is really easy for everyone on sharing and also not and also on localhost.

I done my suggestion now the ball is to you. Many thanks for this amazing software, work and support.

**Comment by imorland** [comment]
Closing as stale — this issue is many years old (predates the 2.x docs and in most cases the current docs structure entirely). The docs have been substantially rewritten since. If this is still a real gap on the current site, please open a fresh issue with up-to-date detail.
